src/AltinnCore/Designer/TypedHttpClients/TypedHttpClientRegistration.cs

```diff
@@ -55,9 +54,8 @@ private static IHttpClientBuilder AddGiteaTypedHttpClient(this IServiceCollectio
             => services.AddHttpClient<IGitea, GiteaAPIWrapper>((sp, httpClient) =>
                 {
                     IHttpContextAccessor httpContextAccessor = sp.GetRequiredService<IHttpContextAccessor>();
-                    IConfigurationSection serviceRepSettings = config.GetSection("ServiceRepositorySettings");
-                    string uriString = serviceRepSettings["ApiEndPoint"];
-                    Uri uri = new Uri(uriString + "/");
+                    ServiceRepositorySettings serviceRepSettings = config.GetSection("ServiceRepositorySettings").Get<ServiceRepositorySettings>();
+                    Uri uri = new Uri(serviceRepSettings.ApiEndPoint + "/");
```

Comment:
It has been defined as a guideline that all urls should end with / in the config file, could you  update appsettings and all other places this value is set to include / and remove + "/" here ?

Reply:
Fixed